src/lib/LibProcessStream.sol

```diff
@@ -17,8 +19,8 @@ library LibProcessStream {
 
     /// Struct representing relevant data for a route.
     struct RouteProcessorData{
```

Comment:
should we rename this struct as it is changing here?

Reply:
Updated